engine/step/condition.go

```diff
@@ -113,6 +123,16 @@ func (a *Assert) Valid() error {
 	return nil
 }
 
+func matchList(valStr, expStr string) bool {
+	values := strings.Split(expStr, ",")
+	for _, v := range values {
+		if valStr == v {
```

Comment:
I'm always paranoid about whitespaces messing up comparisons.
Should we TrimSpace(v) ?

Reply:
If complexity is a problem, we could have a Prepare() step on resolution load